webpack/assets/javascripts/react_app/components/ForemanModal/ForemanModalNoRedux.test.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ForemanModal from '.';
+import ForemanModalHeader from './subcomponents/ForemanModalHeader';
+import ForemanModalFooter from './subcomponents/ForemanModalFooter';
+import { testComponentSnapshotsWithFixtures } from '../../common/testHelpers';
+
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn(fn =>
```

Comment:
can we move the hooks mocking mechanism to `testsHelpers`? to make it easier for reusing?

Reply:
@amirfefer It would be nice but I don't think so, because `jest.mock` only works for the file it's called in.

Reply:
Also the specific hooks that you need to mock may be different for each file.